client/rpcserver/handlers.go

```diff
@@ -452,4 +461,71 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + fmt.Sprintf(feePaidStr, "[fee]") + `"`,
 	},
+	exchangesRoute: {
+		pwArgsShort: ``,
+		argsShort:   ``,
+		cmdSummary:  `List assets supported by all registered DEX servers their and active trades.`,
+		pwArgsLong:  ``,
+		argsLong:    ``,
+		returns: `Returns:
+    map: The exchanges result.
+    {
+      "[DEX url] (map): {
```

Comment:
I'm not sure that `(map)` or `(array)` add any clarity, since they are shown in the structure of the documentation itself. Also missing a closing double-quote on this line.